R/spanning_headers.R

```diff
@@ -21,16 +21,16 @@
 #'   )
 #'
 #' spanning_headers(column_metadata)
-
 spanning_headers <- function(column_metadata) {
-
-  span_element <- function(x) (
-    c("", stringr::str_replace_na(x, ""))
-  )
+  span_element <- function(x) {
+    (
```

Comment:
if we're altering this function we should dtrop these weird superfluous parentheses....

Reply:
I think it's mainly my editor, which automatically styles everything I open and save. Fixed anyway.